i18n/en.json

```diff
@@ -8,5 +8,8 @@
   "exportModalIncludeRelated": "Include related documents",
   "exportModalIncludeChildren": "Include children of this page",
   "exportModalIncludeRelatedSettings": "Related Documents Settings",
-  "exportModalNoRelatedTypes": "No Related Types"
+  "exportModalNoRelatedTypes": "No Related Types",
+  "import": "Import",
+  "importModalDescription": "Importing content requires a zip file. See our <a href=\"https://v3.docs.apostrophecms.org/\" target=\"_blank\">guide to importing content</a> in Apostrophe.",
```

Comment:
I suppose it will redirect to the proper doc later that doesn't exist already?
We might create a ticket for that in order to not forget , wdyt?

Reply:
yes, I'll a note to the existing ticket that's related to writing the doc about sharing docs across sites